styles/styles.css

```diff
@@ -459,30 +459,58 @@ main > .section > .grid-container {
 }
 
 .grid-container {
-  max-width: 100%;
+  /* flexbox fallback */
+  display: flex;
+  gap: 1rem;
 
   @media (min-width: 22.5rem) {
-    /* flexbox fallback */
-    display: flex;
     flex-wrap: wrap;
   }
 
   @supports (display: grid) {
-    @media (min-width: 22.5rem) {
-      display: grid;
-      grid-template-columns: repeat(8, [col] 1fr);
-      grid-column-gap: 4rem;
-      grid-row-gap: 4rem;
-    }
+    display: grid;
+    max-width: 100%;
+    grid-template-columns: repeat(8, [col] 1fr);
+    grid-column-gap: 1rem;
+    grid-row-gap: 1rem;
 
     @media (min-width: 48rem) {
       grid-template-columns: repeat(12, [col] 1fr);
+      grid-column-gap: 1.5rem;
+      grid-row-gap: 1.5rem;
+    }
+  }
+}
+
+.grid-container--large-gap {
```

Comment:
`--mariana-trench` `--grand-canyon` `--tsangpo-gorge` are much more creative, but I'll allow this one to stand